upgrade-scripts/postgresql/rhpam-7.13-to-7.14.sql

```diff
@@ -0,0 +1,2 @@
+alter table if exists migration_reports
```

Comment:
There won't be 7.14 release, so I would just update current file https://github.com/kiegroup/process-migration-service/blob/main/upgrade-scripts/postgresql/rhpam-7.12-to-7.13.sql to include these changes when migrating to 7.13 version

Reply:
Done